crates/storage/db/src/implementation/mdbx/cursor.rs

```diff
@@ -204,6 +204,9 @@ impl<'tx, K: TransactionKind, T: DupSort> DbDupCursorRO<'tx, T> for Cursor<'tx,
 impl<'tx, T: Table> DbCursorRW<'tx, T> for Cursor<'tx, RW, T> {
     /// Database operation that will update an existing row if a specified value already
     /// exists in a table, and insert a new row if the specified value doesn't already exist
+    ///
+    /// For a DUPSORT table, if the key already exists, the data value will be added to the given
+    /// key. Even if the subkeys are the same.
```

Comment:
as someone who doesn't have detailed knowledge about dupsort, this is hard to understand.

> the data value will be added to the given key

does this mean the same thing what's stated in the doc block above? "Database operation that will update an existing row if a specified value already"

I don't really get what's different compared to the docs above

Reply:
yeh see my new comment